com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.cs

```diff
@@ -441,6 +459,7 @@ internal byte GetNeighbourhoodValidityMask(int probeIdx)
         internal class CellInfo : IComparable<CellInfo>
         {
             public Cell cell;
+            public BlendingCellInfo blendingCell;
```

Comment:
For future: have quicksort accept a lambda comparison function so we can avoid having a separate blendingCellInfo class